app/Kpi/FirstReplyKpi.php

```diff
@@ -2,14 +2,19 @@
 
 namespace App\Kpi;
 
-
-class FirstReplyKpi extends Kpi{
-
+class FirstReplyKpi extends Kpi
+{
     const KPI = Kpi::KPI_FIRST_REPLY;
 
-    public static function doesApply($ticket, $comment){
-        if( ! $comment->user) return false;
-        if ( $ticket->comments()->whereNotNull('user_id')->count() > 1 ) return false;
+    public static function doesApply($ticket, $comment)
+    {
+        if (! $comment->user) {
+            return false;
+        }
+        if ($ticket->comments()->whereNotNull('user_id')->count() > 1) {
```

Comment:
Maybe be more verbose here as well. What are you checking? `$ticket->hasReply()`, maybe?